app/src/debug/java/net/squanchy/navigation/DebugActivityIntentFactory.kt

```diff
@@ -0,0 +1,11 @@
+package net.squanchy.navigation
+
+import android.content.Context
+import android.content.Intent
+
+import net.squanchy.support.debug.DebugActivity
+
+class DebugActivityIntentFactory {
+
+    fun createDebugActivityIntent(context: Context) = Intent(context, DebugActivity::class.java)
```

Comment:
Can't this be a top-level function?

Reply:
No, I actually thought about it but we have both a debug and release version of this class, that's why it needs to be a class